packages/elements/src/radio-button/__test__/radio-button.test.js

```diff
@@ -447,6 +448,43 @@ describe('radio-button/RadioButton', () => {
       const checkedRadio = group.find(element => element.checked);
       expect(checkedRadio.id).to.equal('btn2');
     });
+    it('Should separate scope between shadow dom and light dom', async () => {
```

Comment:
```suggestion
    it('Should have separate registry when scope is between shadow DOM and light DOM', async () => {
```

Reply:
I thought we're not going to split the registry? The registry is the same but when matching, we check the scope.

Reply:
Oops sorry.

Reply:
```suggestion
    it('Should separate scope between shadow DOM and light DOM', async () => {
```